setup.cfg

```diff
@@ -34,9 +28,7 @@ classifiers =
 
 [options]
```

Comment:
We also need https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires